src/asGrid/index.js

```diff
@@ -8,17 +8,26 @@ import type {
 } from '../types'
 import { styles, getCol } from './grid.styles'
 import { getValue } from '../utils'
-import asCore from '../core/asCore'
+import { sharedResolutionProperties, getCoreStyles } from '../core'
```

Comment:
there should be only a single default export from `core`. The goal is that you can either `import { x } from 'gymnast` or `import x from 'gymnast/x'` and they are both equivalent.

This allows a consumer to potentially use the [babel lodash plugin](https://github.com/lodash/babel-plugin-lodash) to reduce the size of gymnast even further.

If these values are meant to be exported they should be available as `gymnast/getCoreStyles` and if not they should live elsewhere (e.g. `gymnast/core/utils`).

Because `core` is not exported, this wasn't caught by [these tests](https://github.com/gymnastjs/gymnast/blob/master/test/e2e/gymnast.spec.js#L16) but it may be useful to keep the file structure consistent